src/PowerSystems2PRAS.jl

```diff
@@ -862,6 +886,65 @@ function generate_pras_system(
     end
 end
 
+const DEFAULT_DEVICE_MODELS = [
+    DevicePRASModel(PSY.ThermalGen, GeneratorFormulation),
+    DevicePRASModel(PSY.RenewableGen, GeneratorFormulation),
+    DevicePRASModel(PSY.HydroDispatch, GeneratorFormulation),
+    DevicePRASModel(PSY.Storage, StorageFormulation),
+    DevicePRASModel(PSY.HybridSystem, HybridSystemFormulation),
+    DevicePRASModel(PSY.HydroEnergyReservoir, HydroEnergyReservoirFormulation),
+]
+
+const _LUMPED_RENEWABLE_DEVICE_MODELS = [
+    DevicePRASModel(PSY.ThermalGen, GeneratorFormulation),
+    DevicePRASModel(PSY.RenewableGen, GeneratorFormulation, lump_renewable_generation=true),
+    DevicePRASModel(PSY.HydroDispatch, GeneratorFormulation),
+    DevicePRASModel(PSY.Storage, StorageFormulation),
+    DevicePRASModel(PSY.HybridSystem, HybridSystemFormulation),
+    DevicePRASModel(PSY.HydroEnergyReservoir, HydroEnergyReservoirFormulation),
+]
+
+const DEFAULT_TEMPLATE = PRASProblemTemplate(PSY.Area, DEFAULT_DEVICE_MODELS)
+
+"""
+    $(TYPEDSIGNATURES)
+
+Sienna/Data PowerSystems.jl System is the input and an object of PRAS SystemModel is returned.
+...
+
+# Arguments
+
+  - `sys::PSY.System`: Sienna/Data PowerSystems.jl System
+  - `aggregation<:PSY.AggregationTopology`: "PSY.Area" (or) "PSY.LoadZone" {Optional}
+  - `lump_region_renewable_gens::Bool`: Whether to lumps PV and Wind generators in a region because usually these generators don't have FOR data {Optional}
+  - `export_location::String`: Export location of the .pras file
+    ...
+
+# Returns
+
+    - `PRASCore.SystemModel`: PRAS SystemModel object
+
+# Examples
+
+```julia-repl
+julia> generate_pras_system(psy_sys, PSY.Area)
+PRAS SystemModel
+```
+"""
+function generate_pras_system(
```

Comment:
Is this here to preserve backwards comapatability?

Reply:
Yes.